tests/stress/stress_test.py

```diff
@@ -161,8 +161,16 @@ def _heartbeat(self):
     def _test_large_bundle(self):
         self._set_worksheet('large_bundles')
         large_file = TestFile('large_file', self._args.large_file_size_gb * 1000)
-        self._run_bundle([self._cl, 'upload', large_file.name()])
+        dependency_uuid = self._run_bundle([self._cl, 'upload', large_file.name()])
```

Comment:
Can you document that this needs to be larger than the max memory on the system to test that things are being streamed?

Reply:
Also, one way to make tests run faster is to impose artificial memory limits on the process.